runner/server_plan.go

```diff
@@ -134,6 +135,15 @@ func (r *TerraformRunnerServer) Plan(ctx context.Context, req *PlanRequest) (*Pl
 		planOpt = append(planOpt, tfexec.Target(target))
 	}
 
+	for _, path := range r.terraform.Spec.TfVarsFiles {
+		secureTfVarsFile, err := securejoin.SecureJoin(req.SourceRefRootDir, path)
```

Comment:
Can we add a test where the referenced file doesn't exist.

There can be two routes if the file is not there:
1. We let terraform fail. If a file is not there, it should fail, but if terraform doesn't fail for some reasons, that can be an unexpected behaviour.
2. We check the file in this loop and return with an error if the file is not there. This one would be maybe better as we can give back a meaningful error message without other clutter from the command output.

In any cases, that would be good to see a test that fails intentional because of a missing referenced file.

What do you think?

Reply:
I agree. Will do.

Reply:
@yitsushi fixes + added a new test for non existing file